akka-stream/src/main/scala/akka/stream/Graph.scala

```diff
@@ -69,6 +70,15 @@ trait Graph[+S <: Shape, +M] {
   def addAttributes(attr: Attributes): Graph[S, M] = withAttributes(traversalBuilder.attributes and attr)
 }
 
+object Graph {
+  def mapMaterializedValue[S <: Shape, M1, M2](g: Graph[S, M1])(f: M1 => M2): Graph[S, M2] = {
```

Comment:
Scaladoc like on the other `mapMaterializedValue`s on both of these, perhaps also mention this is Java API and the implicit one Scala API?

Reply:
Minor: no braces needed here.